examples/quantization_aware_training/torch/anomalib/main.py

```diff
@@ -44,11 +44,7 @@ def download_and_extract(root: Path, info: download.DownloadInfo) -> None:
     downloaded_file_path = root / info.url.split("/")[-1]
     print(f"Downloading the {info.name} dataset.")
     with download.DownloadProgressBar(unit="B", unit_scale=True, miniters=1, desc=info.name) as progress_bar:
-        urlretrieve(
-            url=f"{info.url}",
-            filename=downloaded_file_path,
-            reporthook=progress_bar.update_to,
-        )
+        urlretrieve(url=f"{info.url}", filename=downloaded_file_path, reporthook=progress_bar.update_to)  # nosec
```

Comment:
Why can't we use `download_and_extract` as in [OpenVINO example](https://github.com/openvinotoolkit/nncf/blob/develop/examples/post_training_quantization/openvino/anomaly_stfpm_quantize_with_accuracy_control/main.py?)? Why should we use `urlretrieve`?

Reply:
Are there any other options instead of just putting `nosec`?

Reply:
`urlretrieve` is in the black list of bandit https://bandit.readthedocs.io/en/1.7.2/blacklists/blacklist_calls.html, as far I understand, the risk of using `urlretrieve` is in allowing use of ‘file:’’ or custom schemes, but the example uses the hard code url and cann't be specified by user.